Source/Assets/PlayFabEditorExtensions/Editor/Scripts/Utils/PlayFabEditorHelper.cs

```diff
@@ -42,6 +43,7 @@ public static partial class PlayFabEditorHelper
             { DEBUG_REQUEST_TIMING, new PfDefineFlag { Flag = DEBUG_REQUEST_TIMING, Label = "ENABLE REQUEST TIMES", Category = PfDefineFlag.FlagCategory.Feature, isInverted = false, isSafe = true } },
             { ENABLE_BETA_FETURES, new PfDefineFlag { Flag = ENABLE_BETA_FETURES, Label = "ENABLE UNSTABLE FEATURES", Category = PfDefineFlag.FlagCategory.Feature, isInverted = false, isSafe = true } },
             { ENABLE_PLAYFABPLAYSTREAM_API, new PfDefineFlag { Flag = ENABLE_PLAYFABPLAYSTREAM_API, Label = "ENABLE SIGNALR", Category = PfDefineFlag.FlagCategory.Feature, isInverted = false, isSafe = false } },
```

Comment:
Let's delete ENABLE_PLAYFABPLAYSTREAM_API (just here) because that's the old SignalR stuff that we're deprecating.